src/audible_cli/cmds/cmd_download.py

```diff
@@ -641,7 +641,17 @@ def display_counter():
 @click.option(
     "--title", "-t",
     multiple=True,
-    help="tile of the audiobook (partial search)"
+    help="title of the audiobook (partial search)"
+)
+@click.option(
+    "--author", "-a",
```

Comment:
The `-a` option is already used by `--asin`.

Reply:
Oops =] I thought I'd looked for that.

All things considered, I think the more natural assumption would be to have the different lists merged before checking -- I just haven't had time to come back to this since then and I'm not sure how soon I can

Reply:
I think I'll have time this weekend to rework some things and the merge your request.